tracer/test/test-applications/integrations/Samples.GoogleProtobuf/sample.proto

```diff
@@ -0,0 +1,30 @@
+// if modified, run `protoc --csharp_out=. sample.proto` in this folder
+syntax = "proto3";
+package sample;
+
+import "google/protobuf/timestamp.proto";
+
+message Person {
+  string name = 1;
+  string email = 3;
+  repeated PhoneNumber phones = 4;
+  google.protobuf.Timestamp created = 5;
+  google.protobuf.Timestamp last_updated = 6;
+  
+  enum PhoneType {
+    UNSPECIFIED = 0;
+    MOBILE = 1;
+    HOME = 2;
+    WORK = 3;
+  }
+
+  message PhoneNumber {
```

Comment:
How feasible would it be to test the path where we hit the limits for the number of properties and/or the extraction depth?

Reply:
number of properties idk, I think it's something that we'd rarely encounter, and I'm not even sure my code is 100% aligned with what Java does.
The extraction depth is tested indirectly in the protobuf exploration tests, and I think we should test it in a system test, not sure it adds a lot of value to _also_ test it here